python/benchmark/benchmark/bench_kmeans.py

```diff
@@ -192,17 +192,6 @@ def gpu_cache_df(df: DataFrame) -> DataFrame:
 
             cluster_centers = gpu_model.cluster_centers_
 
-            # temporary patch for DB with spark-rapids plugin
-            # this part is not timed so overhead is not critical, but should be reverted
-            # once https://github.com/NVIDIA/spark-rapids/issues/10770 is fixed
-            db_version = os.environ.get("DATABRICKS_RUNTIME_VERSION")
-            if db_version:
-                dim = len(cluster_centers[0])
-                # inject unsupported expr (slice) that is essentially a noop
-                df_for_scoring = df_for_scoring.select(
-                    F.slice(feature_col, 1, dim).alias(feature_col), output_col
-                )
-
```

Comment:
Removing since it looks resolved: https://github.com/NVIDIA/spark-rapids/issues/10770

Reply:
Good find.